### ISSUE TITLE
Threads with unread messages are auto-expanded

### ISSUE THREAD

**Issue by RalfJung** [protz/GMail-Conversation-View#1006]
In the first-run dialogue, I accepted the offer to configure Thunderbird as preferred by Conversations. The only change I made since then is to go back to the "All" folders view, as the unified view moves all these extra IMAP folders some of my mail gets sorted into waaay down to the bottom.

Now, when I go to a folder which has an unread message, Thunderbird expands the thread with that message. This is undesirable with Conversations (with conversations being the unit of manipulation, I don't think I ever want threads to be expanded), and the first-run assistant actually claimed that it changed the settings to prevent this. Somehow, that doesn't seem to work.

I am using Icedove (Thunderbird in Debian) 38.1.

**Comment by protz** [comment]
Yeah. There used to be a hidden setting (mail.scroll_to_unread IIRC) that would prevent this behavior but apparently, this is now something that you can no longer tweak (it's a core Thunderbird behavior, something an addon has no control over). In Edit > Preferences > Advanced > General > Config Editor, please double-check that you have `mailnews.scroll_to_new_message` set to false.

**Comment by RalfJung** [comment]
It is indeed set to `false`.

**Comment by protz** [comment]
What about this one? Can you file on the bugzilla tracker a bug about Thunderbird and CC :protz ?

**Comment by RalfJung** [comment]
Done, see <https://bugzilla.mozilla.org/show_bug.cgi?id=1214534>
